### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by whitneykidd**
Updated formatting on view files
Updated tests for sad paths/edge cases
SimpleCov is at 100%

### CODE REVIEW

File: app/controllers/apps_controller.rb [hunk 1]
```diff
@@ -35,17 +31,12 @@ def update
...
     pet_app = PetApplication.find_by(pet_id: params[:pet_id], application_id: params[:id])
-  
-    if pet.pet_applications.any? { |app| app.approved == true }
-      flash[:notice] = "No more applications can be approved for this pet at this time."
-      redirect_to "/applications/#{@app.id}"
-    else
-      pet.update(status: "Pending")
-      pet_app = pet.pet_applications.find_by(application_id: params[:id])
-      pet_app.update(approved: true)
-      redirect_to "/pets/#{pet.id}"
```
Comment (whitneykidd) [inline app/controllers/apps_controller.rb:46]: This was commented out from our last to final attempt at this method

File: app/controllers/apps_controller.rb [hunk 2]
```diff
@@ -4,10 +4,6 @@ def new
     @favorites = favorite.favorite_pets
   end
 
-  def index
```
Comment (janegreene) [inline app/controllers/apps_controller.rb:7]: unused method removed

File: app/controllers/pets_controller.rb [hunk 3]
```diff
@@ -49,24 +49,6 @@ def destroy
     redirect_to "/pets" 
   end
 
-  def change_status
```
Comment (janegreene) [inline app/controllers/pets_controller.rb:52]: unused method now removed

File: spec/features/pets/show_spec.rb [hunk 4]
```diff
@@ -50,6 +50,7 @@
     expect(page).to have_content(@pet1.name)
     expect(page).to have_content(@pet1.age)
     expect(page).to have_content(@pet1.sex)
+    expect(page).to have_no_content(@pet2.name)
```
Comment (janegreene) [inline spec/features/pets/show_spec.rb:53]: adding sad paths to older tests
